module-system/sov-modules-macros/tests/dispatch/modules.rs

```diff
@@ -88,3 +88,60 @@ pub mod second_test_module {
         }
     }
 }
+
+pub mod third_test_module {
+    use super::*;
+
+    pub trait ModuleThreeStorable:
+        borsh::BorshSerialize + borsh::BorshDeserialize + core::fmt::Debug + Default
```

Comment:
Question: do we get clear error message if we skip ` borsh::BorshSerialize + borsh::BorshDeserialize ` bounds?

Reply:
Yep, we get a good one!

 ```
 error[E0599]: the method `set` exists for struct `StateValue<OtherGeneric>`, but its trait bounds were not satisfied
 --> tests/dispatch/modules.rs
  |
  |             self.state_in_third_struct.set(&msg, working_set);
  |                                        ^^^ method cannot be called on `StateValue<OtherGeneric>` due to unsatisfied trait bounds
  |
  = note: the following trait bounds were not satisfied:
          `OtherGeneric: BorshSerialize`
          `OtherGeneric: BorshDeserialize`
```